.github/workflows/tests.yml

```diff
@@ -1,38 +1,27 @@
 name: tests ci
-# This workflow runs the pipeline with the minimal test dataset to check that it completes any errors
+# This workflow runs the pipeline with the minimal test dataset to check 
+# is completed without any errors
 on:
-  push:
-    branches: [develop]
-  pull_request_target:
-    branches: [develop]
-  release:
-    types: [published]
+    pull_request:
+    push:
 
 jobs:
-  push_dockerhub:
-    name: Push new Docker image to Docker Hub (dev)
+  test_ci:
+    name: Code testing
     runs-on: ubuntu-latest
-    # Only run for the official repo, for releases and merged PRs
-    if: ${{ github.repository == 'BU-ISCIII/taranis' }}
-    env:
-      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
-      DOCKERHUB_PASS: ${{ secrets.DOCKERHUB_PASSWORD }}
+    
     steps:
       - name: Check out pipeline code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
-      - name: Build new docker image
-        run: docker build --no-cache . -t buisciii/taranis:dev
+      - name: Install dependencies
+        run: $CONDA/bin/conda env update --file environment.yml --name base
 
-      - name: Push Docker image to DockerHub (develop)
-        run: |
-          echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
-          docker push buisciii/taranis:dev
-  run-tests:
-    name: Run tests
-    needs: push_dockerhub
-    runs-on: ubuntu-latest
-    steps:
-      - name: Run pipeline with test data
-        run: |
-            docker run buisciii/taranis:dev bash -c /opt/taranis/test/test.sh
+      # - name: install python packages
+      #   run: pip install -r requirements.txt
+
+      - name: Install taranis
+        run: pip install .
+
+      - name: test analyze schema
+        run: taranis analyze-schema --help
```

Comment:
Add the test! it should work right?

Reply:
test  for analyze schema is working now

Reply:
nice!! 🥳 
i've made a comment suggesting change the conda-env-create to tests, so the name is properly displayed in the PR